installer/win/Symphony-x64.aip

```diff
@@ -174,6 +175,7 @@
     <ROW File="Jobber.exe" Component_="Jobber.exe" FileName="Jobber.exe" Attributes="0" SourcePath="..\..\dist\win-unpacked\resources\app.asar.unpacked\node_modules\spawn-rx\vendor\jobber\Jobber.exe" SelfReg="false" NextFile="keyboardlayoutmanager.node" DigSign="true"/>
     <ROW File="LICENSE.electron.txt" Component_="blink_image_resources_200_percent.pak" FileName="LICENS~1.TXT|LICENSE.electron.txt" Attributes="0" SourcePath="..\..\dist\win-unpacked\LICENSE.electron.txt" SelfReg="false" NextFile="LICENSES.chromium.html"/>
     <ROW File="LICENSES.chromium.html" Component_="blink_image_resources_200_percent.pak" FileName="LICENS~1.HTM|LICENSES.chromium.html" Attributes="0" SourcePath="..\..\dist\win-unpacked\LICENSES.chromium.html" SelfReg="false" NextFile="natives_blob.bin"/>
+    <ROW File="ScreenShareIndicatorFrame.exe" Component_="ScreenShareIndicatorFrame.exe" FileName="SCREEN~2.EXE|ScreenShareIndicatorFrame.exe" Attributes="0" SourcePath="..\..\node_modules\screen-share-indicator-frame\ScreenShareIndicatorFrame.exe" SelfReg="false" DigSign="true"/>
```

Comment:
I think it is better to use the `dist` path for this file instead of from `node_module`

Reply:
I just copied how the ScreenSnippet works (next line) as the indicator frame is the same type of thing. Does this mean ScreenSnippet is wrong as well? Also, it seems to work fine - what is the advantage of "dist path"?

Reply:
@mattias-symphony We can use either one of the paths, but `dist` represents the distributable folder, hence we choose that. You are right that ScreenSnippet.exe is also pointed to `node_modules`, we'll need to fix that. If you see the x86.aip file, we point everything to the `dist` folder, so, we'll benefit from being consistent in the two files. I suspect that ScreenSnippet.exe might be accidentally pointing to `node_modules` instead of `dist` in this case.

Reply:
Ok, so I change both to use dist path

Reply:
@keerthi16 @VishwasShashidhar Actually, this will not work - it looks like for the ScreenSnippet tool, it is not actually copied to the dist folder, it only lives in the node_modules, where it is build. So I guess that is why the installer points to node_modules for that one as well. So any suggestions how we would fix this?

Reply:
@mattias-symphony It's available in `dist\win-unpacked\resources\app.asar.unpacked\node_modules\screen-snippet\bin\Release\ScreenSnippet.exe`

Reply:
Ah, yes, it is there :) I'll change it to use that